common/djangoapps/nodebb/signals/handlers.py

```diff
@@ -9,10 +9,20 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from student.models import ENROLL_STATUS_CHANGE, EnrollStatusChange
 from xmodule.modulestore.django import modulestore
+from custom_settings.models import CustomSettings
 
 log = getLogger(__name__)
 
 
+@receiver(post_save, sender=CourseOverview, dispatch_uid="nodebb.signals.handlers.set_is_featured_false")
+def set_is_featured_false(sender, instance, created, **kwargs):
+    if created:
+        course_key = instance.id
+        CustomSettings.objects.get_or_create(course_key=course_key, defaults={'is_featured': False})
```

Comment:
nit: we can omit `defaults` here since `is_featured` defaults to false in model.

Reply:
Done